src/logger.ts

```diff
@@ -1,4 +1,4 @@
-import * as PrettyError from "pretty-error";
+import PrettyError from "pretty-error";
```

Comment:
Had to do this after upgrading `@kubernetes/client-node`.

Reply:
But upgrading `@kubernetes/client-node` doesn't improve the situation vis-a-vis its transitive dependency that has some issues, so I'm going to not both with that and then this change also isn't needed.